src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1353,20 +1386,47 @@ def _configure_security_policy_storage_params(arg_ctx):
             ])
 
         c.argument('administrator_login',
-                   required=True)
+                   required=False)
 
         c.argument('administrator_login_password',
-                   required=True)
+                   required=False)
 
         c.argument('assign_identity',
                    options_list=['--assign-identity', '-i'],
                    help='Generate and assign an Azure Active Directory Identity for this server '
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('enable_ad_only_auth',
+                   options_list=['--enable-ad-only-auth'],
```

Comment:
boolean or only support true?

Reply:
we support only true in server create, null and false have same behavior. the same has been checked in in powershell as well.